SingularityUI/app/views/dashboard.coffee

```diff
@@ -119,4 +122,43 @@ class DashboardView extends View
     changeUser: =>
         app.deployUserPrompt()
 
+    getRequest: (id) =>
+        maybeRequest = @collection.models.filter (model) ->
+            model.id is id
+        if maybeRequest
+            return maybeRequest[0]
+        else
+            return
+
+    viewJson: (e) ->
+        id = $(e.target).parents('tr').data 'request-id'
+        request = @getRequest id
+        unless request
+            console.error ("Could not find request #{id}. Perhaps someone removed it?")
```

Comment:
A console.error would hide this error from anyone without the console open. To most it would look like it just wasn't working and they might keep clicking. Let's keep this message, but use something like `Messenger().error` instead (see application.coffee for an example). This will give a little dispersible message telling the user went wrong.